Resources/config/security.xml

```diff
@@ -17,8 +18,13 @@
         <service id="Sensio\Bundle\FrameworkExtraBundle\Security\ExpressionLanguage" class="Sensio\Bundle\FrameworkExtraBundle\Security\ExpressionLanguage" public="false" />
 
         <service id="Sensio\Bundle\FrameworkExtraBundle\EventListener\IsGrantedListener" class="Sensio\Bundle\FrameworkExtraBundle\EventListener\IsGrantedListener" public="false">
+            <argument type="service" id="Sensio\Bundle\FrameworkExtraBundle\Request\ArgumentNameConverter" />
             <argument type="service" id="security.authorization_checker" on-invalid="null" />
             <tag name="kernel.event_subscriber" />
         </service>
+
+        <service id="Sensio\Bundle\FrameworkExtraBundle\Request\ArgumentNameConverter" class="Sensio\Bundle\FrameworkExtraBundle\Request\ArgumentNameConverter" public="false">
```

Comment:
no need to specify the class, as it is equal to the id (and Symfony 3.3 is the min version)